### DISCUSSION TITLE
week 10 project update

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#201] [General]
<img width="531" alt="Screenshot 2023-07-17 104353" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/54e1229d-a275-45d0-8135-183638565f0e">

Here is the link to my repo. I think if you click on the index.html file, you will be able to see my code at least: https://github.com/USU-ITLS-x265-23-summer/Skye404Project/tree/main

**Comment by CJ3315** [comment]
That's awesome that you have created those section headings, those will be a great foundation to build upon. Are you planning to do different sections or keep it on one page?

**Reply by Skye404** [reply]
I was thinking of keeping it all on one page, if that works. I vaguely remember a possible requirement for multiple pages though. I'll have to reread the project info.

**Comment by audreyhki** [comment]
For the graphic design samples, I think it would be awesome if you added them in list form that linked to another page or tab for viewing. Is this going to be your official portfolio website? Or do you have another that has already been in use for your graphic design?
I'm asking because if you have another site with your graphic projects already set up, you could simply link to that.

**Reply by Skye404** [reply]
No, I don't have another website. I was hoping I could create the embedded images, and clicking on them would make them bigger in a preview for easier viewing.

**Comment by CJ3315** [comment]
I think that is a great idea. That is what I was able to do with my Youtube Content: 
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/28637f0f-a952-445c-8d10-b4172f8264ad)
I have thumbnail images and when you hover over them a video title and a clickable icon is displayed and when you click it takes you to Youtube. 
Here is a video that taught me how to insert links that I found very helpful; https://youtu.be/gOioxltfh48
